packages/components/src/docs/General/10-FAQ.mdx

```diff
@@ -0,0 +1,71 @@
+3. 🛠️ Maintainable & reliable
+Visit our [introduction page](https://solid-design-system.fe.union-investment.de/x.x.x/storybook/index.html?path=/docs/docs-general-introduction--docs#principles) for more details.
+
+### How do components, styles, samples, and patterns differ in our library?
+In our library, we offer four distinct types of elements:
+1. Components: These are lightweight [web components](https://developer.mozilla.org/en-US/docs/Web/API/Web_Components) constructed with lit-element.
+2. Styles: These are even lighter CSS classes that can be applied to any HTML element.
+3. Samples: These are practical demonstrations showcasing the usage of our components and styles within specific contexts.
+4. Patterns: These are predefined use-cases designed to address specific design requirements without the need for a dedicated component.
+Visit our [introduction page](https://solid-design-system.fe.union-investment.de/x.x.x/storybook/index.html?path=/docs/docs-general-introduction--docs#structure) for more details.
+
+### Which browsers are supported?
+TBA
+
+### How can I submit a feature request or report a bug?
+To do so, navigate to the issue section on GitHub and utilize our provided issue templates. Whether you're suggesting a new feature or reporting a bug, these templates offer placeholders and designated content sections where you can input the necessary information. Please ensure to include as many detailed specifics as possible when submitting your request or bug report.
+
+### Is there an endpoint to the development of the Solid Design System?
+No, the Solid Design System is an ongoing endeavor, much like the ever-evolving nature of the web itself. We are committed to continually refining and expanding its features, ensuring it remains aligned with the latest standards and technologies.
+
+### Is there a roadmap for upcoming features?
+Sure. You can explore our project board for a detailed overview of our [roadmap](https://github.com/orgs/solid-design-system/projects/1/views/48).
+
+### What we do and what we don't do?
+- ✅ Provide guidance and support with components, styles, samples and patterns
+- ✅ Create, update and maintain our components, styles, samples and patterns
+- ✅ Write documentation about the system’s elements’ behaviours and interactions
+- tba
+
+- ❌ Screen designs
+- ❌ Prototype flows
+- ❌ Specify your product's rules
+- ❌ Maintain business area libraries
+- tba
+
+### Does the use of the Solid Design System guarantee my website's accessibility?
+No, while our components, styles, samples, and patterns are crafted to fulfill WCAG 2.2 specifications independently, they do not ensure complete web accessibility on their own. Our offerings constitute only a portion of your website or application. Your usage, content, and overall structure play significant roles in achieving full accessibility. We aim to provide flexibility while encouraging you to consider these aspects to deliver a fully accessible experience.
```

Comment:
Implementing Solid Components into your product is an important first step in making web applications accessible. But it is not a guarantee.

Every time a component is used on a page, the context and order of the content must be considered. It is important to consider what purpose the component serves (correct and meaningful semantics).

Product teams must take responsibility for ensuring that their digital products based on Solid DS continue to comply with these standards.

Reply:
rephrased it.